js/deer-config.js

```diff
@@ -590,6 +598,14 @@ export default {
                     }
                     queryString = queryString.trim()
                     const query = decodeContentState(queryString)
+                    const queryCopy = {}
```

Comment:
I know this will be frustrating because the code works.  

You do not need a `queryCopy` you are doing too much work.  Simply use the existing `query` object, and only read from it.  Do not clone it, copy it, or change it.